client/landing/stepper/declarative-flow/internals/steps-repository/unified-plans/unified-plans-step.tsx

```diff
@@ -551,7 +595,20 @@ function UnifiedPlansStep( {
 							}
 						/>
 					}
-					heading={ <Step.Heading text={ getHeaderText() } subText={ fallbackSubHeaderText } /> }
+					heading={
+						<>
+							{ ( visualSplitVariation === 'default_websitebuilder' ||
+								visualSplitVariation === 'default_hosting' ) && (
```

Comment:
Nit pick: I see we're checking intent for the variation - would make to check for it here as well?